File: loops.py
# loops

# for,  while

nombres = ["Hugo", "Paco", "Luis"]

for x in range(0, len(nombres), 1): # start (optional), stop, step (optional)
    if x == 1: break
    print(nombres[x])

for nombre in nombres:
    print(nombre)

i = 0
while(i < len(nombres)):
    print(nombres[i])
    i = i + 1   

File: loops.js
// loops

// for, for in, for of, while, do while, 

let nombres = ["Hugo", "Paco", "Luis"];

for(let i=0; i < nombres.length; i ++){
    if (i == 2) break;
    console.log(nombres[i]);
}

for(let i in nombres){
    console.log(nombres[i]);
}

for(let nombre of nombres){
    console.log(nombre);
}

let i = 0;
while(i < nombres.length){
    console.log(nombres[i]);
    i++;
}